test/pkg/kernel-config/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: tag push
 
 BASE=alpine:3.5
-IMAGE=check
+IMAGE=kernel-config
```

Comment:
I think all the other images are prefixed with `test-` and then this should also be prefixed.